apps/src/templates/progress/progressStyles.js

```diff
@@ -64,6 +77,215 @@ export const hoverStyle = {
     'background-color .2s ease-out, border-color .2s ease-out, color .2s ease-out'
 };
 
+/**
+ * ======================================
+ * Bubble styles
+ * ======================================
+ */
+
+export const BubbleSize = makeEnum('dot', 'letter', 'full');
+export const BubbleShape = makeEnum('circle', 'diamond', 'pill');
+
+/**
+ * Note: these constants will be removed in favor of `bubbleSizes` below once
+ * we finish cleaning up all of our bubble components.
+ */
+export const DOT_SIZE = 30;
```

Comment:
this is the "legacy" main bubble size. the scrolling bug ultimately was caused by mistakenly setting `bubbleSizes[BubbleShape.circle]` to this same value (30). 

in several places we rely on `BUBBLE_CONTAINER_WIDTH` to get things to line up properly, which was computed like so: 
```
// Hard-coded container width to ensure all big bubbles have the same width
// regardless of shape (circle vs. diamond).
// Two pixels on each side for margin, plus 2 x border width
export const BUBBLE_CONTAINER_WIDTH = DOT_SIZE + 4 + 2 * BUBBLE_BORDER_WIDTH;
```

as part of the cleanup i switched the bubbles to use `box-sizing: border-box` so we don't have to factor the border width into size calculations. so the container width calculation now only adds the implicit margins (see line 147 below). however, i failed to consider that since the border widths are now included in the bubble size, the bubble size needs to be made 4px larger to remain at the same rendered size as before.

Reply:
Since there are different ways to interpret size, I thought it might be helpful to indicate which "size" these values refer to.  Up to you.  (Also ok to defer to a later PR since we're doing more work here soon.)